easybuild/easyconfigs/l/libfabric/libfabric-2.0.0-GCCcore-14.2.0.eb

```
@@ -0,0 +1,61 @@
+easyblock = 'ConfigureMake'
+
+name = 'libfabric'
+version = '2.0.0'
+
+homepage = 'https://ofiwg.github.io/libfabric/'
+description = """
+Libfabric is a core component of OFI. It is the library that defines and exports
+the user-space API of OFI, and is typically the only software that applications
+deal with directly. It works in conjunction with provider libraries, which are
+often integrated directly into libfabric.
+"""
+
+# The psm3 provider (enabled by default) requires an AVX capable system to run
+toolchain = {'name': 'GCCcore', 'version': '14.2.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/ofiwg/libfabric/releases/download/v%(version)s']
+sources = [SOURCE_TAR_BZ2]
+checksums = [
+    {'libfabric-1.21.0.tar.bz2': '0c1b7b830d9147f661e5d7f359250b85b5a9885c330464cd3b5e5d35b86551c7'},
```

```suggestion
    {'libfabric-2.0.0.tar.bz2': '1a8e40f1f331d6ee2e9ace518c0088a78c8a838968f8601c2b77fd012a7bf0f5'},
```

Still using old checksum. Installation therefore fails with:

```
== FAILED: Installation ended unsuccessfully (build directory: /data/EasyBuild-develop/build/libfabric/2.0.0/GCCcore-14.2.0): build failed (first 300 chars): Missing checksum for libfabric-2.0.0.tar.bz2 in {'libfabric-1.21.0.tar.bz2': '0c1b7b830d9147f661e5d7f359250b85b5a9885c330464cd3b5e5d35b86551c7'} (took 0 secs)
== Results of the build can be found in the log file(s) /tmp/eb-e_j4ibvk/easybuild-libfabric-2.0.0-20250113.114616.Bnuve.log
```

---

I have this filtered so that slipped through sorry, but yes I will mark this PR draft as it's basically waiting for UCX 1.18

---

UCX 1.18.0 is out now: https://github.com/openucx/ucx/releases/tag/v1.18.0